src/utils/notificationsLocalStorage.ts

```diff
@@ -92,6 +94,13 @@ export const setProcessError = (error: any) => {
       subscribeError: errorMessage,
     }),
   );
+  openToast({
+    id: 'notifications-subscription-error',
+    type: 'ERROR',
+    title: "Push notification error",
+    description: `${errorMessage}. Please try again later or send us a message if the problem persists.`,
+    duration: 5000,
+  });
```

Comment:
It's worth to show users that there is something went wrong. At least we'll have a better understanding what should we improve.

I wonder if we should add an analytics event here?

Reply:
Good question - I'll defer to @charleslavon on this one. Seems like a good idea to track it.

Reply:
this error message is helpful 👍🏾 .  We don't need the analytics events, because we have this info via the serverless fn logs